org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/workflowEngine/Workflow.java

```diff
@@ -0,0 +1,95 @@
+package org.eclipse.ice.modeling.workflowEngine;
+
+/**
+ * Workflow is the entity that is responsible for managing the processing of a data set.
+ * It is an abstract class which will be instantiated by a concrete class.  It is part
+ * of composition pattern so clients only work with the Workflow interface and do not
+ * need to know anything about the concrete classes or their interfaces.
+ */
+public class Workflow {
```

Comment:
Add abstract here if Workflow is an abstract class (which I agree it should be)

Reply:
Agree.  I think there are some abstract methods along the way too.  However, my familiarity with using "abstract" particularly with Java and somewhat in general is not real strong.  (Knowledge that has long since fled my brain.)